www/src/pages/showcase.js

```diff
@@ -29,8 +29,8 @@ export const showcaseQuery = graphql`
           childScreenshot {
             screenshotFile {
               childImageSharp {
-                sizes(maxWidth: 512) {
-                  ...GatsbyImageSharpSizes
+                fluid(maxWidth: 512) {
```

Comment:
would love to chat about the difference here at some point

Reply:
They're pretty much the same thing — we just renamed them as "sizes" and "resolutions" were near universally not understood :-D https://next.gatsbyjs.org/docs/migrating-from-v1-to-v2/#rename-responsive-image-queries

Reply:
Totally unrelated, but this is such a good change :) It _always_ confused me which one was which.

Reply:
Glad to hear!